girder/utility/mail_utils.py

```diff
@@ -41,7 +41,7 @@ def getEmailUrlPrefix():
         host = '://'.join((cherrypy.request.scheme,
                            cherrypy.request.local.name))
         if cherrypy.request.local.port != 80:
-            host += ':{}'.format(cherrypy.request.local.port)
+            host += ':%s' % cherrypy.request.local.port
```

Comment:
Fixed, though it shouldn't matter unless you want to do more advanced string formatting inside ``%d``, right?